bodhi/server/util.py

```diff
@@ -604,7 +604,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
+        else:
+            log.error(err)
```

Comment:
Hello @crungehottman!

That ```ValueError``` probably happened because you might have forgotten to add a parameter to the test function to accept the mock object. Unless you provide ```patch``` with an object to replace the object with, you need to make sure the test accepts the mock itself as a parameter. This is also useful because you can use that parameter to inspect the mock. Here's some code to illustrate:

```
@mock.patch('some.thing')
def test_cool_stuff(self, thing):
    """Note that thing is in my parameter list, and can be used to see what calls to thing happened."""
    self.assertTrue(thing is 'cool')
```

Now, you can actually also use patch to set the object to something explicitly, in which case the mock isn't handed to you. This is handy when you don't need to assert that the mock was used in any particular way, but just want to cause some side effect (for example, maybe you just want to change a setting or something.) Here's an example of that style:

```
@mock.patch('some.thing', mock.MagicMock(return_value='cool'))
def test_cool_stuff(self):
    """Note that I didn't get the mock as a parameter this time."""
    self.assertEqual("This test is useless", "This test is useless")
```